video/out/wayland_common.c

```diff
@@ -2722,6 +2767,34 @@ void vo_wayland_handle_scale(struct vo_wayland_state *wl)
                                 lround(mp_rect_h(wl->geometry) / wl->scaling_factor));
 }
 
+bool vo_wayland_valid_format(struct vo_wayland_state *wl, uint32_t drm_format, uint64_t modifier)
+{
+#if HAVE_DRM
+    // Tranches are grouped by preference and the first tranche is at the end of
+    // the list so we loop backwards.
```

Comment:
This comment may be true, but it is a little misleading. In fact, the order of iteration or even the use of multiple tranches currently doesn't make a difference. We try to match all the formats from all the tranches. That's fine, but I would clarify this in the comment or remove it to avoid confusion.

Reply:
Reworded it a bit.